test/gui/shared/scripts/helpers/api/sharing_helper.py

```diff
@@ -11,6 +11,23 @@
     {"user": 0, "group": 1, "public_link": 3, "federated_cloud_share": 6}
 )
 
+PERMISSIONS = {"read": 1, "update": 2, "create": 4, "delete": 8, "share": 16}
+
+
+def getPermissionsRoleIdByName(permissionsRole):
+    permissionList = [perm.strip() for perm in permissionsRole.split(",")]
```

Comment:
let's use snake_case. we will enable case lint in the future
```suggestion
    permission_list = [perm.strip() for perm in permissionsRole.split(",")]
```